src/themes/teams/components/ItemLayout/itemLayoutVariables.ts

```diff
@@ -0,0 +1,30 @@
+import { pxToRem } from '../../../../lib'
+
+export default siteVars => {
+  const vars: any = {}
+
+  // Item
+  vars.itemPaddingLeft = pxToRem(20)
+  vars.itemPaddingRight = pxToRem(18)
+  vars.columnGap = pxToRem(8)
```

Comment:
This looks like a regression.  The listVariables.ts was moved to listItemVariables.ts and the variable names were updated to remove the `list*` portion.

See master:
https://github.com/stardust-ui/react/blob/master/src/themes/teams/components/List/listItemVariables.ts